token_transfer_private.go

```diff
@@ -0,0 +1,39 @@
+package hedera
```

Comment:
Not a big fan of `private` in the name of the file.

Reply:
What would be best for the name, I changed it to internal for now.